SConstruct

```diff
@@ -7,6 +7,16 @@ import tarfile
 import zipfile
 from SCons.Script.SConscript import SConsEnvironment
 
+# Compatibility function for Python 2.6.6
+if "check_output" not in dir(subprocess):
```

Comment:
Actually, this would be marginally better as `if hasattr(subprocess, "check_output"):`